### PR TITLE
Init new project structure

### PR DESCRIPTION
**Pull request by aeniinik**
### What has changed?

Initiliazed the project with the desired libraries. There is an boilerplate example for the features which has routes, unittest and e2e tests.

There's an example for redux.

### Why was the change made?

To initialize the project with the decided techs.

### Caveats?

Readme needs attention in the future.

Features need to have:
- api

### Related Trello issue
[Set up new repo](https://trello.com/c/MBeExiza/326-set-up-new-repo)
### Checklist
- [ ] I have updated relevant documentation in READMES

### CODE REVIEW

**Comment by aeniinik** [comment]
I'll add githooks and redux.

**Review by rottabonus — APPROVED** [review]
Great readme =)

What i tested: ran the project, wrote some code and committed - all works as expected

Only thing that comes to my mind now is that we could add .nvmrc or some other way to specify the node-version we are using :P It can be done on a separate pr too

File: src/features/RegisterPage/index.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import RegisterPage from './RegisterPage';
+
+export default RegisterPage;
```
Comment (Enzine) [inline src/features/RegisterPage/index.ts:3]: This seems like an unnecessary file. Now there are two files exporting RegisterPage. Exporting it in and importing from the RegisterPage.tsx should be sufficient.
```suggestion
```

File: src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { Routes, Route, Link } from 'react-router-dom';
+import styled from 'styled-components';
+import RegisterPage from '@/features/RegisterPage';
+
+const Wrapper = styled.section``;
```
[the remark this answers is not in the document]
Comment (aeniinik) [reply src/App.tsx:5]: This is just boilerplate to check that styled-components are installed.

File: src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+function App() {
```
[the remark this answers is not in the document]
Comment (aeniinik) [reply src/App.tsx:7]: I personally find function in some cases more readable. But it's good topic to discuss when to use arrow functions and when the function keyword. Functions support hoisting, which might be the best reason to use them with React.

**Comment by aeniinik** [comment]
I'll merge this into master. Some notes by @Enzine are good to address in the future. I'll make a Trello issue.

File: src/features/RegisterPage/index.ts (hunk 1 above)
Comment (Enzine) [reply src/features/RegisterPage/index.ts:3]: Could everything be in the index.tsx then?

**Pull request merged**